native/src/test/resources/packages/test-consolidate/Ballerina.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+org = "asmaj"
+name = "test_consolidate"
+version = "0.1.0"
+distribution = "2201.10.1"
```

Comment:
@azinneera Should we use a newer version?

Reply:
Fixed in https://github.com/ballerina-platform/consolidate-packages-tool/commit/bd473123358220cca26188349581184775fe47ca to use U12